samples/csharp_dotnetcore/16.proactive-messages/Startup.cs

```diff
@@ -21,6 +21,9 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
+            // Create the credential provider to be used in the NotifyController.
+            services.AddSingleton<ICredentialProvider, ConfigurationCredentialProvider>();
```

Comment:
if this is using the overload of the constructor:

```
protected BotFrameworkHttpAdapter(IConfiguration configuration, ILogger<BotFrameworkHttpAdapter> logger = null)
```

Then we shouldn't need the explicit IConfigurationProvider on the services collection???